src/Persistent/Api.hs

```diff
@@ -128,11 +129,20 @@ dbTopics = dbTopicMap . to Map.elems
 emptyAulaData :: AulaData
 emptyAulaData = AulaData nil nil nil nil nil 21 21 30 3 0
 
--- FIXME move enough specialized calls to IO in PersistM to remove MonadIO
-class MonadIO m => PersistM m where
+class Monad m => PersistM m where
     getDb :: AulaGetter a -> m a
     modifyDb :: AulaSetter a -> (a -> a) -> m ()
 
+    getCurrentTimestamp :: m Timestamp
+
+    default getCurrentTimestamp :: MonadIO m => m Timestamp
+    getCurrentTimestamp = Timestamp <$> liftIO getCurrentTime
```

Comment:
Perfect. This mention of MonadIO doesn't leak any IO power outside, but documents that we mean the ordinary IO stuff.